groovy/material/FirstDegreeMaterials.groovy

```diff
@@ -3472,7 +3472,7 @@ public class FirstDegreeMaterials {
         Kovar = new Material.Builder(8619, SuSyUtility.susyId('kovar'))
                 .ingot()
                 .iconSet(SHINY)
-                .flags(GENERATE_ROD, GENERATE_LONG_ROD, GENERATE_RING, GENERATE_PLATE, DISABLE_DECOMPOSITION)
```

Comment:
this allows the player to transmute cobalt matte into pure cobalt, dont do this

Reply:
do you have any alternative idea on how to get the cobalt rod for cobalt sulfate electrolysis to get the 1st pure cobalt dust then